zone/npc.cpp

```diff
@@ -3239,6 +3239,18 @@ void NPC::AIYellForHelp(Mob *sender, Mob *attacker)
 		}
 
 		float assist_range = (mob->GetAssistRange() * mob->GetAssistRange());
+
+		if (RuleB(Combat, EnableSneakPull) && attacker->sneaking) {
+			assist_range=(20*20);
```

Comment:
Formatting, could just put 400 being it's hard-coded anyway or make it a rule maybe

Others below need spacing in declaration.

Reply:
Done and thx.